tests/test_discover.py

```diff
@@ -110,7 +112,28 @@ def test_episodes_for_subtopic(self, mock_get):
     self.assertEquals(episode_ids, ids)
 
   # No ML endpoints
+  def test_series_for_user_no_ml(self):
+    config.ML_ENABLED = False
+    response1 = self.user1.get('api/v1/discover/series/user/?offset=0&max=1')
+    response2 = self.user1.get('api/v1/discover/series/user/?offset=1&max=1')
+    series1 = json.loads(response1.data)['data']['series'][0]
+    series2 = json.loads(response2.data)['data']['series'][0]
+    self.assertTrue(int(series1['id']) != int(series2['id']))
+    self.assertTrue(series1['subscribers_count'] >= \
```

Comment:
we clear the sub count and rec count between every test, so won't these always be 0?